### ISSUE TITLE
Remediations for 5.2.2 and 5.2.3 incorrectly use chown instead of chmod.

### ISSUE THREAD

**Issue by estenrye** [darkwizard242/cis_ubuntu_2004#4]
# Current Remediation for 5.2.2:
https://github.com/darkwizard242/cis_ubuntu_2004/blob/aaccda1ce3c10e599a16d2d751741d0481441496/defaults/main/section_05.yml#L107

This sets the owner of the key to `0600`

# Remediation in CIS Benchmark for 5.2.2:
```yaml
ubuntu_2004_cis_section5_rule_5_2_2_params_permissions_command: find /etc/ssh -xdev -type f -name 'ssh_host_*_key' -exec chmod u-x,go-rwx {} \; 
```

# Current Remediation for 5.2.3

https://github.com/darkwizard242/cis_ubuntu_2004/blob/aaccda1ce3c10e599a16d2d751741d0481441496/defaults/main/section_05.yml#L113

This sets the owner of the key to `0600`

# Remediation in CIS Benchmark for 5.2.3:
```yaml
ubuntu_2004_cis_section5_rule_5_2_3_params_permissions_command: find /etc/ssh -xdev -type f -name 'ssh_host_*_key.pub' -exec chmod u-x,go-rwx {} \; 
```

**Comment by darkwizard242** [comment]
@estenrye - Thanks very much for reporting this. I missed out on the fact that I mistakenly ended up writing chown instead of chmod to set octal file permissions. 

I am currently working on "CIS Benchmark for Ubuntu 20.04 v1.1.0" in a [feature branch](https://github.com/darkwizard242/cis_ubuntu_2004/tree/feature/cis_version_1.1.0) that will be merged into master upon completion (timeframe: within a month) and will address the fix for this alongside. Also, updated [CHANGELOG.md](https://github.com/darkwizard242/cis_ubuntu_2004/blob/feature/cis_version_1.1.0/CHANGELOG.md) of the branch to match and eventually merge.

**Comment by darkwizard242** [comment]
@estenrye - This has been addressed in Release [2.0.0](https://github.com/darkwizard242/cis_ubuntu_2004/releases/tag/2.0.0)
